autoload/fzf_mru/actions.vim

```diff
@@ -21,6 +21,15 @@ function! fzf_mru#actions#options() abort
   return options
 endfunction
 
+function! s:preview(...)
+  let preview_window = get(g:, 'fzf_preview_window', '')
+  if empty(preview_window)
+    return a:1
+  endif
+  let arguments = add(copy(a:000), preview_window)
```

Comment:
Yep, that would work, as long as `preview_window` is a string. Once its array you will end up with arguments like this:
`[{'options': ['--layout=reverse', '--info=inline']}, ['right:50%', 'ctrl-/']]`
and that when passed to `fzf#vim#with_preview` will get us a monster:
`{'options': ['--layout=reverse', '--info=inline', '--preview', 'bat {}', '--bind', '[''right:50%'', ''ctrl-/'']']}`

And according to fzf readme it should always be an array.

> #### Preview window
> 
> Some commands will show the preview window on the right. You can customize the
> behavior with `g:fzf_preview_window`. Here are some examples:
> 
> ```vim
> " This is the default option:
> "   - Preview window on the right with 50% width
> "   - CTRL-/ will toggle preview window.
> " - Note that this array is passed as arguments to fzf#vim#with_preview function.
> " - To learn more about preview window options, see `--preview-window` section of `man fzf`.
> let g:fzf_preview_window = ['right:50%', 'ctrl-/']
> 
> " Preview window on the upper side of the window with 40% height,
> " hidden by default, ctrl-/ to toggle
> let g:fzf_preview_window = ['up:40%:hidden', 'ctrl-/']
> 
> " Empty value to disable preview window altogether
> let g:fzf_preview_window = []
> ```

Looks like it used to be a string, they doing check for this [here](https://github.com/junegunn/fzf.vim/blob/d43df0ea2f099836c2e3499756101e87bcbd3e25/plugin/fzf.vim#L53-L57)

I guess you can do same check here.

Reply:
That would include changing `add` to `extend` and always pass `preview_window` array.

Reply:
How about this?
```sh
let preview_args = get(g:, 'fzf_preview_window', ['right', 'ctrl-/'])
if !empty(preview_args)
  if type(preview_args) == type('')
    let preview_args = [preview_args]
  endif
  let arguments = extend([extra], preview_args)
  let extra = call('fzf#vim#with_preview', arguments)
endif
```

Reply:
@CsYakamoz That's basically what I linked. Still needs to be amended to work with current code, so I'm not sure what this comment is for to be honest?